src/ssl/ServerBump.h

```diff
@@ -51,7 +52,7 @@ class ServerBump
         Ssl::BumpMode step2; ///< The SSL bump mode at step2
         Ssl::BumpMode step3; ///< The SSL bump mode at step3
     } act; ///< bumping actions at various bumping steps
-    Ssl::BumpStep step; ///< The SSL bumping step
+    XactionSteps step; ///< The TLS bumping step
```

Comment:
I am not excited about renaming SslBump to TlsBump because it will introduce a yet another naming inconsistency into the code, but if we have to do it:

```suggestion
    XactionStep step; ///< the current TlsBump step
```

or perhaps

```suggestion
    XactionStep step; ///< the current or last TlsBump step
```

Please consider typedef-ining Ssl::BumpStep to be XactionSteps instead.

Reply:
I undo any changes in Ssl::ServerBump class and I make Ssl::BumpStep a typedef of XactionStep instead.

commit 953af56adc